src/routes/blog/post/password-protection/+page.markdoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Password protection for developers: importance and best practices"
```

Comment:
Remove quotes?

Reply:
Or is it needed because you have the `:` in the title?

Reply:
Yes, it is needed because of the `:`